models/intel/smartlab-object-detection-0004/README.md

```diff
@@ -0,0 +1,61 @@
+# smartlab-object-detection-0004
+
+## Use Case and High-Level Description
+
+This is a smartlab object detector that is based on YoloX for 416x416 resolution.
+
+## Example
+
+![](./assets/frame0001.jpg)
+
+## Specification
+
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP @ [ IoU=0.50:0.95 ]          | see PASCAL per-cls AP (internal test set) |
```

Comment:
iou 0.5:0.95 is usually coco metric, not voc, some misleading here...

Reply:
also you marked as resolved, but no any changes were provided. metric still named as 
AP @ [ IoU=0.50:0.95 ], which is really incorrect naming
and if it is possible, please keep metric directly in this table, it is hard to read and logically strange, when you use separated table for metric right below the main table